### DISCUSSION TITLE
バッジの詳細を開いたときのモーダルがウインドウサイズによっては見切れる

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#528] [要望・アイデア・不具合]
[獲得したバッジ一覧ページ](https://qiita.com/badges)において、バッジの詳細を開いたときのモーダルがウインドウサイズによっては見切れます。

<img width="1552" alt="スクリーンショット 2023-08-25 15 27 59" src="https://github.com/increments/qiita-discussions/assets/20086673/e00b1216-9ae6-48e9-a427-0b447bb824c7">

上記のスクリーンショットは13インチMacBook ProでChromeを最大化した場合の表示であり、極端にディスプレイが小さい環境というわけではありません。
モーダルの縦幅を小さくするか、スクロールできるようになっているといいかと思います。

**Comment by tanaka-asami415** [comment]
@munierujp 
ご報告ありがとうございます！🙇
修正いたします！

**Reply by tanaka-asami415** [reply]
@munierujp 
こちら修正してスクロールできるようにいたしました！
ご報告いただきありがとうございました！🙇

![image (6)](https://github.com/increments/qiita-discussions/assets/103916185/ed011dbc-e397-4328-ae5f-e99d376e2798)
